dist/index.js

```diff
@@ -18768,7 +18769,14 @@ async function retrieveToken(method, client) {
             const githubAudience = core.getInput('jwtGithubAudience', { required: false });
 
             if (!privateKey) {
-                jwt = await core.getIDToken(githubAudience)
+                try {
+                    core.debug("Fetching ID token from GitHub OIDC provider")
+                    jwt = await core.getIDToken(githubAudience)
+                } catch (err) {
+                    const new_error = new Error(`Error fetching ID token from GitHub OIDC provider., message: ${err.message}`);
```

Comment:
Is there a reason this is a const?

Reply:
Leftover habit from coding in typescript at my last job.
const and let are generally best practices for javascript over var ever since they were introduced in es6: https://www.freecodecamp.org/news/differences-between-var-let-const-javascript/
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

Basically, since I'm using `const` here, another error object can't get assigned to the `new_error` variable. You can still update properties of the object, but the original object pointer can't change.